src/features/ai/settings/OpenAiSettingsPane.tsx

```diff
@@ -124,3 +114,14 @@ export function OpenAiSettingsPane({ config }: SettingsPaneProps) {
     </SettingsPane>
   );
 }
+
+function ChatModelTooltip() {
+  return (
+    <div className="text-btn-ico-tool-active">
+      <div id="chat-tooltip">
+        <InfoIcon />
+      </div>
+      {/* TODO: <Tooltip anchorSelect="#chat-tooltip" /> */}
```

Comment:
Should this block the merge?

Reply:
I think it's fine, I'll create a ticket to update the tooltip. The design team is aware of that and is working on the tooltip design and content